haystack/components/routers/zero_shot_text_router.py

```diff
@@ -107,22 +105,23 @@ def __init__(
         huggingface_pipeline_kwargs: Optional[Dict[str, Any]] = None,
     ):
         """
-        Initializes the TransformersZeroShotTextRouter.
+        Initializes the TransformersZeroShotTextRouter component.
 
-        :param labels: The set of possible class labels to classify each sequence into. Can be a single label,
+        :param labels: The set of labels to use for classification. Can be a single label,
             a string of comma-separated labels, or a list of labels.
-        :param multi_label: Whether or not multiple candidate labels can be true.
-            If False, the scores are normalized such that the sum of the label likelihoods for each sequence is 1.
-            If True, the labels are considered independent and probabilities are normalized for each candidate by
+        :param multi_label:
+            Indicates if multiple labels can be true.
+            If `False`, label scores are normalized so their sum equals 1 for each sequence.
+            If `True`, the labels are considered independent and probabilities are normalized for each candidate by
```

Comment:
Very much struggling with this, not sure if we can improve this sentence in any way :D